lib/compilers/sway.ts

```diff
@@ -0,0 +1,245 @@
+import path from 'path';
+
+import fsExtra from 'fs-extra';
```

Comment:
In every other file we:

```suggestion
import fs from 'fs-extra';
```

...and then use the raw `fs` (which has equivalent functionality). Can you change to that please?